data/mods/desert_region/mapgen/waterbody.json

```diff
@@ -0,0 +1,44 @@
+[
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [
+      "lake_bed",
+      "lake_shore",
+      "river_center",
```

Comment:
I believe you could make the dried out rivers prettier by adding a nested mapgen of a blotch of clay or recesses, with a low chance of some puddles in the middle, only spawning in the rivers center. Same for lakes perhaps. Then making river or lake shores mostly mudcracks, sand and dirt with maybe a bit more than usual vegetation.

Reply:
I think the simplistic river centers look *okay* for now, I am more concerned about the overly rectangular shorelines. Turning the shore into a mixture of sand, dirt, and mudcrack would not fix the harsh seams between either the center and the shore or the shore and the land, which is my main issue.

Reply:
It's a problem for another day then.  Thank you for your suggestion, I'll consider it.

Reply:
Nevermind, just realized else_chunks exist.  I'll make a commit here once I'm done.